src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -69,6 +69,7 @@ namespace
             m_scale_multiplier = 1.0f;
 
             m_rendering_threads = 0;    // 0 = as many as there are logical cores
+			m_use_embree = true;
```

Comment:
We want Embree to be _disabled_ by default as it's still quite experimental and doesn't provide much performance benefits yet.

We use 4 spaces for indentation instead of tabs in all appleseed code bases. Replace tabs by spaces everywhere.

Reply:
Agree, I have set Embree to disabled by default.